src/btree/bt_sync.c

```diff
@@ -341,6 +354,17 @@ __wt_sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
                     btree->rec_max_txn = mod->rec_max_txn;
                 if (mod != NULL && btree->rec_max_timestamp < mod->rec_max_timestamp)
                     btree->rec_max_timestamp = mod->rec_max_timestamp;
+
+                /*
+                 * Check whether the tree is lookaside btree and verify whether the page contents
+                 * are obsolete or not by checking the visibility of the page stop time pair.
```

Comment:
I would end the sentence after `are obsolete`. The detail of the page stop time pair is not in this code here and could become outdated and wrong.

Reply:
Fixed.